src/index.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import CalculatorPage from './pages/CalculatorPage';
+import QuotePage from './pages/QuotePage';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="/calculator" element={<CalculatorPage />} />
+        <Route path="/quote" element={<QuotePage />} />
```

Comment:
Well done ✔ The website contains Home, Calculator, and Quote 🔥 All links are working as expected 🙌

Reply:
Routes and its Navigation links implementation are splendid 😉